Polyfrost/Hytils-Reborn
# improve pit lag hider

## Description

This was suggested like 2 or 3 years ago but it never happened. This adds support for hiding nametags and Levelhead from the entities while using Pit Lag Reducer

Might be a lot of code duplication but welcome to Hytils. if have suggestions on how to cleanup pls tell

## Related Issue(s)

sk1ercord suggestion

## How to test

Turn on Pit Lag Reducer
Jump down and see if nametags and levelhead stop rendering

## Release Notes

```release-note
Add support for hiding nametags and levelheads when using Pit Lag Reducer
```

## Documentation
